src/Tribe/Templates.php

```diff
@@ -543,8 +543,8 @@ public static function showInLoops( $query ) {
 		 **/
 		public static function getTemplateHierarchy( $template, $args = array() ) {
 			if ( ! is_array( $args ) ) {
+				$passed        = array( $template, $args );
```

Comment:
This does not seem to be backwards compatible with the `$backwards_map` usage below where it still supports other arguments coming from `func_get_args()`